.babelrc

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
The webpack command didn't seem to pick up the plugins mentioned here for some reason, I moved this config to the webpack.config.js file and it worked. Maybe there is something that I was missing, @MonkeyDo ?